src/components/policy/src/cache_manager.cc

```diff
@@ -755,8 +757,7 @@ void CacheManager::CheckSnapshotInitialization() {
   rpc::Optional<policy_table::UsageAndErrorCounts>& usage_and_error_counts =
       snapshot_->policy_table.usage_and_error_counts;
 
-  if (usage_and_error_counts->is_initialized() &&
-      usage_and_error_counts->app_level->is_initialized()) {
```

Comment:
@vlantonov I assume that `app_level` is optional field and could be uninitialized. So when you dereference it on the next line it could crash isn't it?